scripts/add-cluster.sh

```diff
@@ -64,7 +74,7 @@ while test $# -gt 0; do
             *)
                echo "$1 is not a recognized flag!"
                user_help
-               exit -1
+               exit 0
```

Comment:
Why? It should return non-zero value when there is provided unsupported flag

Reply:
Addressed in d4f3d0d